docs/_user_guide/best_practices_monitor_usage.md

```diff
@@ -1,116 +1,101 @@
+    We have [Community Guidelines](/_policies/user_limits){target="_blank"} to illustrate responsible use of our shared resources that our users are expected to follow. These guidelines ensure that the Yens can remain accessible and performant.
 
-## Other page
-##  Monitoring Usage
-## Monitoring Your Resource Footprint
+## Monitoring Compute with `htop`
+One easy method of getting a quick snapshot of your CPU and memory usage is via the `htop` command line tool. Running `htop` shows usage graphs and a process list that is sortable by user, top CPU, top RAM, and other metrics. Please use this tool liberally to monitor your resource usage, especially if you are running multiprocessing code on shared systems for the first time.
 
-Certain parts of the GSB research computing infrastructure provide
-environments that are managed by a scheduler (like <a href="_policies/services" target="_blank">Sherlock</a> or <a href="/yen/scheduler.html" target="_blank">Yen-Slurm</a>). In these cases it is not necessary for individuals to monitor resource usage themselves.
+You can toggle between a threaded view of processes by pressing ++t++ -- here is sample `htop` output in threaded mode.
 
-However, when working on systems like the <a href="/yen/index.html" target="_blank">interactive yens</a> where resources like **CPU**, **RAM**, and **disk space** are _shared_ among many researchers,
- it is important that all users be mindful of how their work impacts the larger community.
+![resource monitoring during parallel run](/assets/images/monitor_htop_r_8_cores.png)
 
-!!! tip
-    When using interactive yens, use the ```htop``` and ```userload``` commands to monitor CPU and RAM usage. Use the ```gsbquota``` command to monitor disk quota.
+!!! warning "Jobs may be automatically killed"
+    Note that jobs that exceed user limits may be terminated automatically to preserve the integrity of the system.
 
-### CPU & RAM
+## User Limits and Multiprocessing
 
-Per our <a href="/yen/community.html" target="_blank">Community Guidelines</a>, CPU usage should always be limited to 48 CPU cores/threads per user at any one time on yen[2-5] and up to 12 CPU cores on yen1.
-Some software (R and RStudio, for example) default to claiming all available cores unless told to do otherwise.
-These defaults should always be overwritten when running R code on the yens. Similarly, when working with multiprocessing code in languages like Python,
-care must be taken to ensure your code does not grab everything it sees. Please refer to our parallel processing <a href="/topicGuides/index.html" target="_blank">Topic Guides</a> for information about how to limit resource consumption when using common packages.
+Per our [Community Guidelines](/_policies/user_limits){target="_blank"}, CPU usage should always be limited based on system size. When working with implicitly or explicitly *multiprocessed* code, care must be taken to ensure your code does not use every available processor.
```

Comment:
We don't call that page Community Guidelines anymore. So, maybe we refer to it as "User Limits"